examples/elevation/components/elevation/elevation.tsx

```diff
@@ -0,0 +1,61 @@
+import {useEffect, useState} from 'react';
+import {
+  useElevationService,
+  useGoogleMap
+} from '@ubilabs/google-maps-react-hooks';
+
+const Elevation = () => {
+  const {map} = useGoogleMap();
+  // Get the elevator from the useElevationService hook
+  const elevator = useElevationService();
+
+  const [infoWindow, setInfoWindow] = useState<google.maps.InfoWindow | null>(
+    null
+  );
+
+  useEffect(() => {
+    if (!map) {
+      return;
+    }
+
+    if (!infoWindow) {
+      // Create a new InfoWindow
+      const initialPosition = {lat: 51.08998021141488, lng: 10.627828045134935};
+      const infowindow = new google.maps.InfoWindow({
+        content: 'Click somewhere on the map to see the elevation',
+        position: initialPosition
+      });
+
+      setInfoWindow(infowindow);
+      map.setCenter(initialPosition);
+
+      return;
```

Comment:
Why do you stop here, wait for the info window state to be updated and then continue this effect? Couldn't you just do this all in one run?

Reply:
Yes sure